### ISSUE TITLE
Request for test/evaluation script to reproduce results

### ISSUE THREAD

**Issue by LaoKing233** [ssssmark/AesR1#4]
Hi! Thank you for sharing the paper and code.
To help with reproducing the results, could you please provide a test or evaluation script? Thanks again!

**Comment by LaoKing233** [comment]
I used the inference script from LLaMA-Factory to run evaluations on the test sets of all datasets except PARA, and observed a significant gap in performance compared to the results reported in the paper. To better understand and reproduce the official results, could you kindly share the exact evaluation/test script used in the experiments? Thank you!

**Comment by song630** [comment]
@LaoKing233 Thank you so much for providing the details! I will try and share my results with you once I have them.

**Comment by ssssmark** [comment]
@LaoKing233  

Thank you for your attention and interest of our work!

The released huggingface model was trained on the combined training set of 15K samples randomly collected from the official train set of AVA, TAD66K and FLICKR-AES in a 2:2:1 ratio. As we mentioned in our paper, for evaluation, we follow the official test splits for TAD66K, AVA, and PARA; for datasets lacking official splits(Flickr,AADB), we randomly partition the data into train and test sets in a 9:1 ratio using a fixed seed 42 to ensure reproducibility. We provide the vllm inference script using LLaMA-Factory’s scripts/vllm_infer.py.  The performance can be slighly different due to the prompt template you use, the different hardware settings(e.g. inference engine)  , you can copy the prompt in our released dataset for testing

Best regards

**Comment by LaoKing233** [comment]
@ssssmark 
Thank you for the detailed clarification on the dataset splits and evaluation setup! I’ll re-run the inference using the provided details (including the prompt template and split strategy) and report back if I have further questions. Really appreciate your help!
